libtiledbvcf/cmake/Modules/FindHTSlib.cmake

```diff
@@ -79,30 +79,105 @@ if (NOT HTSLIB_FOUND)
     # required to updated htslib configure.ac with autoconf 2.70
     #   - see https://github.com/samtools/htslib/commit/680c0b8ef0ff133d3b572abc80fe66fc2ea965f0
     #   - and https://github.com/samtools/htslib/pull/1198/commits/6821fc8ed88706e9282b561e74dfa45dac4d74c8
+    if(NOT WIN32)
     find_program(AUTORECONF NAMES autoreconf REQUIRED)
+    find_program(AUTOHEADER NAMES autoheader REQUIRED)
+    find_program(BASH_PATH NAMES bash REQUIRED)
+    endif()
 
-    ExternalProject_Add(ep_htslib
-      PREFIX "externals"
-      URL "https://github.com/samtools/htslib/releases/download/1.15.1/htslib-1.15.1.tar.bz2"
-      URL_HASH SHA1=e7cbd4bb059020c9486facc028f750ec0fb2e182
-      UPDATE_COMMAND ""
-      CONFIGURE_COMMAND
-          autoheader
-        COMMAND
-          ${AUTORECONF} -i
+    if(WIN32)
```

Comment:
Please remove all of this.

Reply:
Ok, realizing that doing so without implementing a build against the fetched version leaves no way to build the windows version via this cmake script set within tiledb.
As of today I have the fetch implemented locally, but per our conversation will remove that off to the side for now.

Reply:
update - 'by now' this is replaced by code that fetches JB's msys2 prebuilt htslib.